src/console_window.cpp

```diff
@@ -167,6 +171,11 @@ ConsoleWindow::ConsoleWindow(LogDatabase *db)
     ui.excludeText, SIGNAL(textChanged(const QString &)),
     this, SLOT(excludeFilterUpdated(const QString &)));
 
+  //Connect 'Search' text modification to searchFilterUpdated, VCM 13 April 2017
```

Comment:
Changes incorporated.  Is there a style guide that you use and if so can you send it to me?